sw/device/tests/example_test_from_flash.c

```diff
@@ -70,6 +70,14 @@ const test_config_t kTestConfig = {
 // void ottf_timer_isr(void) {}
 // void ottf_external_isr(void) {}
 
+/**
+ * Place data in flash that will need to persist across resets by marking with
+ * with the section ".non_volatile_scratch"). Write to this region with the
+ * flash controller DIF.
+ */
```

Comment:
should we also say read to this section with `mmio`?

Reply:
Done.

Reply:
You'd actually use `abs_mmio_read32(...)` or similar. :)

Reply:
Fixed.